Wacom tablet basic integration

aspaNeuro opened the pull request:
A first prototype supporting a single tablet device working using the Wintab API:

- Adapted from WintabDN example source code from [here](https://github.com/Wacom-Developer/wacom-device-kit-windows/tree/master/Wintab%20.NET/SampleCode/WintabDN).
- Creates a `WintabCapture` node that hooks to all message events coming from the Wintab API to an invisible window.
- Creates `WintabDataFrame` class from the data packets arriving from Wintab devices.

aspaNeuro committed:
Add reference to windows forms

aspaNeuro committed:
Add original classes from WintabDN

aspaNeuro committed:
Add class to get observable hooked into wintab

aspaNeuro committed:
Add bonsai node to get wintab events into editor

jfrazao committed:
Add wintab licence file

jfrazao committed:
Add NOTICE

jfrazao committed:
Fix wintab typo

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/bonsai-rx/wintab?pullRequest=1) <br/>All committers have signed the CLA.

glopesdev committed:
Fix file casing

glopesdev committed:
Replace nested LICENSE with root NOTICE file

glopesdev committed:
Refactor code to run from WintabCapture

glopesdev committed:
Remove all unnecessary windows forms dependencies

Also removed all try/catch blocks routing to blocking message boxes as
well as redundant error checks.

glopesdev committed:
Ensure packet event handler is removed

glopesdev committed:
Internalize WintabDN and expose managed data frame

glopesdev committed:
Add default none value for packet status enum
